earth_enterprise/BUILD_RHEL_CentOS.md

```diff
@@ -33,110 +41,139 @@ sudo subscription-manager repos --enable=rhel-6-server-optional-rpms
 sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
 ```
 
-_Note: the EPEL URL above assumes that your RHEL 6 installation is running the latest updates_
+## Install Git
 
-## Install Git  
-    
 Install the system default version:
-    
-```
+
+```bash
 sudo yum install -y git
 ```
-    
-Or install the latest version from the IUS repo (recommended) [ [More Info] ](https://ius.io/GettingStarted/):
-```
+
+Alternatively, install the latest version from the IUS repo (recommended).
+See the [Getting Started](https://ius.io/GettingStarted/) page to find the RPM
+URL for your system. Use this URL below:
+
+```bash
 sudo yum install -y wget
 cd /tmp
-wget https://rhel7.iuscommunity.org/ius-release.rpm
+
+# CentOS 7
+wget https://centos7.iuscommunity.org/ius-release.rpm
 sudo yum install -y ius-release.rpm
 sudo yum install -y git2u-all
-```        
-        
-## Install Git LFS
-   
-Execute the following commands
-    
 ```
-curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh | sudo bash
+
+## Install Git LFS
+
+Execute the following commands:
+
+```bash
+curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh \
+  | sudo bash
 sudo yum install git-lfs
 ```
-    
+
 ## GCC 4.8
 
 ### CentOS 7 and RHEL 7
 
-```
-yum install ant bzip2 doxygen gcc-c++ patch python-argparse python-setuptools swig tar
+```bash
+yum install ant bzip2 doxygen gcc-c++ patch python-argparse python-setuptools \
+  swig tar
 ```
 
+### CentOS 6
 
-### CentOS 6 
-```
-yum install -y devtoolset-2-gcc devtoolset-2-binutils devtoolset-2-toolchain devtoolset-2-gcc-gfortran 
+```bash
+yum install -y devtoolset-2-gcc devtoolset-2-binutils devtoolset-2-toolchain devtoolset-2-gcc-gfortran
 ```
 
 ### RHEL 6
 
-```
-sudo yum install -y devtoolset-2-gcc devtoolset-2-binutils  devtoolset-2-toolchain devtoolset-2-gcc-c++ devtoolset-2-gcc-gfortran 
+```bash
+sudo yum install -y devtoolset-2-gcc devtoolset-2-binutils \
+  devtoolset-2-toolchain devtoolset-2-gcc-c++ devtoolset-2-gcc-gfortran
 ```
 
-The GCC 4.8 installation will be located in the `/opt/rh/devtoolset-2/root/usr/bin/` directory.  
+The GCC 4.8 installation will be located in the _/opt/rh/devtoolset-2/root/usr/bin/_.
 
-The GEE build scripts will detect this compiler automatically. However, if you wish switch the environment to use GCC 4.8 in a shell, run: 
-```source /opt/rh/devtoolset-2/enable```
+The GEE build scripts will detect this compiler automatically. However, if you
+wish switch the environment to use GCC 4.8 in a shell, execute:
```

Comment:
Minor: You don't have to fix this, since it's probably not from any of your commits.

Looks like there's a missing preposition: 'wish switch' -> 'wish to switch'.

However, why is our documentation talking about switch the compiler in a shell?  `scons` ignores the shell environment.  This could only be useful if someone is going to compile object files by hand in a shell.  I think it should be removed.